GVFS/GVFS/CommandLine/DehydrateVerb.cs

```diff
@@ -334,12 +349,13 @@ private void SendDehydrateMessage(ITracer tracer, GVFSEnlistment enlistment, str
             {
                 foreach (string folder in response.SuccessfulFolders)
                 {
-                    this.WriteMessage(tracer, $"{folder} folder successfully dehydrated.");
+                    this.WriteMessage(tracer, $"{folder} folder {this.DisplayName} successful.");
                 }
 
                 foreach (string folder in response.FailedFolders)
                 {
-                    this.WriteMessage(tracer, $"{folder} folder failed to dehydrate. You may need to reset the working directory by deleting {folder}, running `git reset --hard`, and retry the dehydrate.");
+                    this.WriteMessage(tracer, $"{folder} folder failed to {this.DisplayName}. You may need to reset the working directory by deleting {folder}, running `git reset --hard`, and retry the {this.DisplayName}.");
+                    folderErrors.Add(folder);

```

Comment:
Is `\0{ioError}` included in the entries in `response.FailedFolders`?

If not, should we an a `\0<error>` here?

Reply:
These folders come back from the mount process and I wasn't sure how a `\0` would be handled by our named pipe messages but let me take a look and try it out.

Reply:
Basic question, is the `\0<error>` just to provide more details to the caller (in case they care about the specific error reason)?

I wasn't sure if callers would expect every line to have a `\0<error>` even if it was just ` `\0{DisplayName} Failed` (or some other general failure message for this case)

Reply:
Yeah we should always have at least the `\0` even if there isn't anything after so that it is an expected format where the `\0` is the field separator.